### PR TITLE
App release process

### PR DESCRIPTION
**Pull request by lavanya-bmw**

### CODE REVIEW

File: cx-portal-shared-components/src/components/basic/Dropzone/components/DropPreview.tsx [hunk 1]
```diff
@@ -86,42 +99,65 @@ export const DropPreview: FunctionComponent<DropPreviewProps> = ({
     DropPreviewFileComponent = () => null
   }
 
+  const onCallback = (closeOverlay: boolean) => {
+    if (closeOverlay) {
+      onDelete?.(deletestatus.index)
+      setDeleteStatus({ index: 0, state: false })
+    } else setDeleteStatus({ index: 0, state: false })
```
Comment (oyo) [inline cx-portal-shared-components/src/components/basic/Dropzone/components/DropPreview.tsx:106]: this is the same call in `if` and `else` - remove the `else` and put this call after the `if`

File: cx-portal-shared-components/src/components/basic/Dropzone/components/DropPreview.tsx [hunk 2]
```diff
@@ -86,42 +99,65 @@ export const DropPreview: FunctionComponent<DropPreviewProps> = ({
...
+      <Box sx={{ marginTop: 4 }}>
+        <DropStatusHeaderComponent
+          numUploaded={uploadedFilesCount}
+          numTotal={filesCount}
+        />
+        {finishedFilesCount > 0 && (
+          <Box sx={{ marginTop: 4 }}>
+            {uploadFiles.map(
+              (file, index) =>
+                isFinished(file) && (
```
Comment (oyo) [inline cx-portal-shared-components/src/components/basic/Dropzone/components/DropPreview.tsx:125]: better use a .filter here and then a .map to iterate over the results
`uploadFiles.filter(isFinished).map(....)`

File: cx-portal-shared-components/src/components/basic/Dropzone/components/DropPreview.tsx [hunk 3]
```diff
@@ -86,42 +99,65 @@ export const DropPreview: FunctionComponent<DropPreviewProps> = ({
...
+                  />
+                )
+            )}
+          </Box>
+        )}
+        {filesCount - finishedFilesCount > 0 && (
+          <Box sx={{ marginTop: 4 }}>
+            {uploadFiles.map(
+              (file, index) =>
+                !isFinished(file) && (
```
Comment (oyo) [inline cx-portal-shared-components/src/components/basic/Dropzone/components/DropPreview.tsx:145]: better use a .filter here and then a .map to iterate over the results
`uploadFiles.filter(isNotFinished).map(....)`

File: cx-portal-shared-components/src/components/basic/Dropzone/components/DropPreviewFile.tsx [hunk 4]
```diff
@@ -98,7 +100,9 @@ export const DropPreviewFile: FunctionComponent<DropPreviewFileProps> = ({
   }
 
   const showDeleteButton =
-    uploadFile.status !== 'uploading' && uploadFile.status !== 'upload_error'
+    enableDeleteIcon &&
+    uploadFile.status !== 'uploading' &&
+    uploadFile.status !== 'upload_error'
```
Comment (oyo) [inline cx-portal-shared-components/src/components/basic/Dropzone/components/DropPreviewFile.tsx:105]: this has been here before but better add a string enum for all the different states. This is more safe than comparing strings.
`enum UploadStatus {`
`   UPLOADING = 'uploading'`
`...`

`uploadFile.status !== UploadStatus.UPLOADING` 
etc...

**Review by oyo — CHANGES_REQUESTED** [review]
some minor suggestions

**Review by oyo — APPROVED** [review]
looks good now

File: cx-portal-shared-components/src/components/basic/Dropzone/components/DropPreview.tsx (hunk 1 above)
Comment (lavanya-bmw) [reply cx-portal-shared-components/src/components/basic/Dropzone/components/DropPreview.tsx]: Resolved

File: cx-portal-shared-components/src/components/basic/Dropzone/components/DropPreview.tsx (hunk 2 above)
Comment (lavanya-bmw) [reply cx-portal-shared-components/src/components/basic/Dropzone/components/DropPreview.tsx]: Resolved

File: cx-portal-shared-components/src/components/basic/Dropzone/components/DropPreview.tsx (hunk 3 above)
Comment (lavanya-bmw) [reply cx-portal-shared-components/src/components/basic/Dropzone/components/DropPreview.tsx]: Resolved

File: cx-portal-shared-components/src/components/basic/Dropzone/components/DropPreviewFile.tsx (hunk 4 above)
Comment (lavanya-bmw) [reply cx-portal-shared-components/src/components/basic/Dropzone/components/DropPreviewFile.tsx]: Resolved

**Pull request merged**